lib/generated/browserSessionReplay.ts

```diff
@@ -0,0 +1,636 @@
+/* eslint-disable */
```

Comment:
<!-- Warning represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 warning:** remove `lib/generated/*` and adjust the `generated` folder path in gitignore